src/test/java/Homework19_20.java

```diff
@@ -17,6 +17,7 @@ public void deletePlaylist () throws InterruptedException {
         clickDeletePlaylistBtn();
         String actualMsg = getDeletedPlaylistMsg();
         String expectedMsg = "Deleted playlist \"Test Pro Playlist.\"";
+        Thread.sleep(5000);
```

Comment:
**Please, delete Thread. sleep**

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
I don't know where you are getting that screenshot because I deleted thread.sleep a long time ago. Here is my screen.
![Screenshot 2023-07-28 at 9 57 56 AM](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/137566317/ccb98e3d-1425-41c6-b0c1-3683dde2b211)

Reply:
Hello

Don't worry! Please, re-submit homework 20 for review

Reply:
I have no changes to make, so nothing to push, and no new pull request to give you.

Reply:
> I have no changes to make, so nothing to push, and no new pull request to give you.

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/c90e35f7-5247-44ad-bfe7-bf5547a19115)